plugin.rb

```diff
@@ -211,6 +211,73 @@ def limit_accepts
   TopicView.add_post_custom_fields_whitelister do |user|
     ["is_accepted_answer"]
   end
+  
+  def before_head_close_meta(controller)
+    return "" if not controller.instance_of? TopicsController
+    topic = Topic.with_deleted.where(id: controller.params[:topic_id]).first
+    return "" if not controller.guardian.allow_accepted_answers_on_category?(topic.category_id)
+    first_post = topic.first_post
+    question_json =  {
+      '@type' => 'Question',
+      'name' => topic.title,
+      'text' => first_post.raw,
+      'upvoteCount' => first_post.like_count,
+      'answerCount' => topic.reply_count,
+      'dateCreated' => first_post.created_at,
+      'author' => {
+        '@type' => 'Person',
+        'name' => first_post.user.username
+      }
+    }
+    page_json = {
+      '@type' => 'QAPage',
+      'name' => topic.title,
+    }
+    accepted_answer_post_id = topic.custom_fields["accepted_answer_post_id"]
+    if accepted_answer_post_id
+      accepted_answer = Post.where(id: accepted_answer_post_id).first
+      question_json[:acceptedAnswer] = {
+        '@type' => 'Answer',
+        'text' => accepted_answer.raw,
+        'upvoteCount' => accepted_answer.like_count,
+        'dateCreated' => accepted_answer.created_at,
+        'url' => accepted_answer.full_url,
+        'author' => {
+          '@type' => 'Person',
+          'name' => accepted_answer.user.username
+        }
+      }
+    end
+    suggested_answer = Post.where(topic_id: topic.id, post_number: controller.params[:post_number]).first
+    if suggested_answer and suggested_answer.post_number != 1
```

Comment:
```suggestion
    if suggested_answer && suggested_answer.post_number != 1
```

and is strange in ruby has weird precedence we always use && in conditionals